src/aesim_config.erl

```diff
@@ -127,6 +130,18 @@ convert(Type, Key, Value)
   catch
     _:badarg -> error({bad_option, Key, Value})
   end;
+convert(number, Key, Value) ->
+  try
+    list_to_integer(Value)
+  catch
+    _:badarg ->
+      try
+        list_to_float(Value)
```

Comment:
Is usage of float affecting simulation determinism? Shall simulation be deterministic? Shall float be avoided by big integers?

Reply:
floats are only used for verified pool selection probability and eviction skew. I don't see it strongly affecting determinism. There is a lot of randomnesses involved anyway. If we want strong determinism maybe we should enforce a seed.